gatsby-remark-oembed/.eslintrc

```diff
@@ -1,13 +1,12 @@
 {
   "env": {
+    "es6": true,
     "node": true,
-    "browser": true,
```

Comment:
The plugin is only run in Node so the browser environment is not necessary. The "commonjs" environment is either invalid or not necessary. I'm not sure which.